README.md

```diff
@@ -2,6 +2,34 @@
 
  Allows use of plain strings and files as credentials to be passed to miscellaneous build steps. See also this [plugin's wiki page][wiki]
 
+## Description
+
+This plugin provides two new types of credentials:
+
+-   `FileCredentials` or *Secret file*:
+
+![](https://wiki.jenkins.io/download/attachments/69763266/secretFile.png?version=1&modificationDate=1463752202000&api=v2){.confluence-embedded-image
```

Comment:
Don’t link to the wiki for images include them in the GitHub repo

Reply:
Yeah Added them to the repo